<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADDSTUDENTWINDOW</class>
 <widget class="QMainWindow" name="ADDSTUDENTWINDOW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>427</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>427</width>
    <height>180</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Add Student Form</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="genderLabel">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>138</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QComboBox" name="genderField">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>136</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Others</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="doneAdd">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>23</y>
      <width>381</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="idLabel">
       <property name="text">
        <string>ID Number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="idField">
       <property name="placeholderText">
        <string>e.g.&quot;2020-1971&quot;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="courseField">
       <property name="placeholderText">
        <string>e.g. &quot;BSCS&quot;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Full Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="yearLabel">
       <property name="text">
        <string>Year Level</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameField">
       <property name="placeholderText">
        <string>e.g. &quot;Jason Dela Cruz</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="courseLabel">
       <property name="text">
        <string>Course</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="yearField">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>idField</tabstop>
  <tabstop>nameField</tabstop>
  <tabstop>courseField</tabstop>
  <tabstop>genderField</tabstop>
  <tabstop>doneAdd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
